docs/dev/README.md

```diff
@@ -63,46 +63,27 @@ compatible agents; but keep in mind _these are optional for humans_ even though
 
 ## release process
 
-> [!TIP]
-> Requires `cargo-release` and `git-cliff` to be installed.
-
 > [!NOTE]
-> In order to have your changes integrate with the changelog,
-> make sure to merge commits to `main` with a meaningful commit message.
+> Make sure to merge commits to `main` with meaningful commit messages.
+> Using [conventional commits](https://www.conventionalcommits.org) format
+> is recommended for better organization of changes in the GitHub release notes.
 >
-> Additionally, if those commits are parseable as [conventional commits](https://www.conventionalcommits.org),
-> they will be grouped into sections in the changelog.
-> See the `cliff.toml` file for more details and group examples.
+> PRs where the title contains the literal `chore` are ignored in the release notes.
 
-Use `cargo release` to create a release.
-Since we cannot push to `main` directly, perform the steps below:
+To create a new release, simply create and push a tag:
 
 ```shell
-# Choose a version. It should be valid semver.
-# Also, choose a branch name. A good default is `prep/$VERSION`.
-VERSION=<VERSION>
-BRANCH="prep/$VERSION"
-
-# Make a branch for release prep and check it out.
-git checkout -b $BRANCH
-
-# Have cargo-release create the release.
-# This does several things:
-# - Validates that the git index is clean
-# - Updates version numbers in the crates
-# - Generates the changelog using `git-cliff`
-# - Creates a commit with the changes
-# - Pushes the branch to the remote
-cargo release --no-publish --no-tag --allow-branch=$BRANCH $VERSION
-
-# Open a PR; once tests pass and reviewers approve, merge to main and come back here for the final step.
-# The PR title should be "chore(release): $VERSION" or something that similarly avoids generating a changelog entry;
-# see the `cliff.toml` file for more details.
-gh pr create --base main --body-file .github/release_template.md --title "chore(release): $VERSION"
-
-# Finally, run `cargo release` on the main branch.
-# This doesn't create new commits; it just tags the commit and pushes the tag.
+# Create and push the tag from main branch
 git checkout main
-git pull
-cargo release -x
+git pull && git pull --tags --force
+git tag v0.5.0 # replace this with the version you want to release
```

Comment:
This should be an annotated tag, right? `--annotate`

Reply:
Non-annotated tags are kind of like branches where they just kind of point at something where an annotated one is [a more fully fledged](https://git-scm.com/book/en/v2/Git-Basics-Tagging) object in the Git history. So it has a date, who tagged it, etc. I agree that the message doesn't really matter.

I think for our purposes it's probably not critical, but the annotated tags seem more correct/harder to mess up to me.

Reply:
🤔 hmm, ok, i'll update the docs. i think everywhere else for other repos we use non-annotated tags though so that would likely be worth bringing up since you have this opinion

Reply:
I guess maybe it doesn't matter. I thought we used annotated tags mostly because the CLI's process [documents](https://github.com/fossas/fossa-cli/blob/master/docs/contributing/releases.md?plain=1#L11) it that way so I've always used annotated tags. 

But it looks like BMO just uses lightweight ones, so I guess I'm the odd one out. If we've been getting on without it for this long I guess it's not too important.